### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by arcuri82**

### CODE REVIEW

File: docs/publications.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Publications
 
+The development of _EvoMaster_ is rooted in academia.
+Here, you can find the PDFs of all the academic publications based on _EvoMaster_. 
+Furthermore, slides of presentations can be  found [here](publications.md).
```
Comment (ZhangMan1126) [inline docs/publications.md]: presentations.md ?

Comment (arcuri82) [reply docs/publications.md]: you are right. thx! fixed now

**Comment by jgaleotti** [comment]
Apologies for the delayed comment.

Regarding writer_driver.md I would suggest the following to improve the organisation.
- separate documents for embedded and external controllers (recommending embedded as the easier driver to write)
- an additional document describing how to deal with authentication (so, if no authentication is needed, the reader does not need to know that) 
- an additional document describing changes needed to add database (i.e. resetStateOfSUT, P6Spy, etc). 

Then, the organisation could be: 
- create driver for embedded springboot with no DB, no authentication
- changes for authentication (if needed)
- changes fot DB (if needed)

IMHO, these changes are a nice-to-have, but they are not mandatory.

**Comment by arcuri82** [comment]
thx! I added those feedback on Trello, as not much time now to do major changes / restructuring now :(

**Pull request merged**